src/parser.rs

```diff
@@ -1,33 +1,31 @@
-use nom::{
-    bytes::complete::{tag, take_till},
-    character::complete::space0,
-    combinator::map,
-    multi::{many0, many1, separated_list1},
-    sequence::pair,
-    sequence::terminated,
-    IResult,
-};
+pub fn parse_file(input: &str) -> Vec<usize> {
+    if input.len() == 0 {
+        vec![]
+    } else {
+        let mut result = vec![];
+        let mut whitespace = 0;
+        let mut non_space = false;
```

Comment:
I struggled to say "not non-space" while reading. What do you think of calling this `space = true` instead?